contactme.css

```diff
@@ -89,43 +85,40 @@
   border: #fff 1px solid;
 }
 
-.imgcontainer-contact{
- margin-top: 3.5rem;
- display: flex;
- justify-content: center;
+.imgcontainer-contact {
+  margin-top: 3.5rem;
+  display: flex;
+  justify-content: center;
 }
 
-@media only screen and (min-width: 720px) {
-
-  .contact-container-flex{
+@media only screen and (min-width: 992px) {
```

Comment:
- [ ] - [ ] Add some padding/margin between these two columns. Do this for each of the projects.
![Screenshot from 2021-08-19 12-54-31](https://user-images.githubusercontent.com/65501535/130048695-8d4aa7cf-6c8c-46b5-ad1c-28c7f608a1f3.png)